dashboard/app/controllers/levels_controller.rb

```diff
@@ -536,24 +536,34 @@ def extra_links
     if @level.level_concept_difficulty && !@level.level_concept_difficulty.concept_difficulties_as_string.empty?
       links[@level.name] << {text: "LCD: #{@level.level_concept_difficulty.concept_difficulties_as_string}", url: ''}
     end
+
+    if params[:scriptLevelId]
+      script_level = ScriptLevel.find_by(id: params[:scriptLevelId].to_i)
+    end
+
     is_standalone_project = ProjectsController::STANDALONE_PROJECTS.values.pluck(:name).include?(@level.name)
+
     # Curriculum writers rarely need to edit STANDALONE_PROJECTS levels, and accidental edits to these levels
     # can be quite disruptive. As a workaround you can navigate directly to the edit url for these levels.
     # We allow editing from the test environment to enable UI testing of the edit page.
     if (Rails.application.config.levelbuilder_mode || rack_env?(:test)) && !is_standalone_project
       can_edit_level = can? :edit, @level
+
```

Comment:
This is me just breaking up a bunch of if statements for readability, can revert if ppl don't like :)